### DISCUSSION TITLE
How to notify our users on new posts in our blog?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#132] [Ideas]
Hi,

Following the discussion today in the AEDAPT meeting, I wonder how should we notify Neurodesk users (or prospective users who attended a talk and want to stay in the loop) on new posts in the blog. Very naively, I pressed on the RSS button in the blog, and got this:
<img width="1680" alt="Screen Shot 2021-09-24 at 11 44 31 am" src="https://user-images.githubusercontent.com/8280611/134605853-8e01a012-a739-49a1-bb8c-38bcfd0ac254.png">
So it's obviously not a good solution for the average user who does not use a news aggregator like feedly (and possibly doesn't even know what RSS is).

Users can subscribe to our twitter account, but I don't think it's a good medium for important announcement (like new releases) either, because many people (like myself) get many many tweets, and don't get to browse through all of them.

I still think that letting users subscribe for email notifications on blog posts by entering their email address in the website is the best straightforward option. Yes - somebody needs to implement it, but it's well worth the investment.

Any thoughts?

Oren

**Comment by stebo85** [comment]
Dear @civier ,

I think a possibility to get notified via email is very useful. However, as you said: Someone needs to implement it (and potentially maintain the solution later on) :)

So: Go for it! Figure out a way of doing it and propose a way to implement it.


Cheers
Steffen

**Comment by civier** [comment]
That can be a possible implementation of a subscription system as used by ARCOS:
<img width="1680" alt="Screen Shot 2021-09-24 at 12 02 33 pm" src="https://user-images.githubusercontent.com/8280611/134607226-cfbaa663-8214-4cc7-ae7d-b943b5f559c9.png">
I'll do something myself or liaise with somebody.

**Comment by stebo85** [comment]
That would be great! Looking forward to it :)

**Comment by civier** [comment]
@air2310 I will work on an email notification service, but in the meantime, I wonder if you can add links to at least Chrome and Firefox RSS reader plug-ins next to RSS icon? (after installing them, something will actually happen when people press on the RSS icon :-) 
https://chrome.google.com/webstore/detail/rss-feed-reader/
https://addons.mozilla.org/en-US/firefox/addon/feedbroreader/
The Chrome plug-in does not allow getting email notifications (unless you have the paid version, I think), but it does pop up notifications in the browser itself.

I tried to edit the blog page myself, but I got this, and I'm not sure what I should change there exactly:
<img width="1680" alt="Screen Shot 2021-09-24 at 12 35 08 pm" src="https://user-images.githubusercontent.com/8280611/134609915-03e61a16-aef7-4345-b613-3195b09ead36.png">

**Comment by stebo85** [comment]
@civier,

Even with these plugins the RSS experience will not be great. How about we remove the RSS button, then no-one gets confused by it? And you can focus on setting up a mailinglist that people can subscribe to that you manage?

**Comment by civier** [comment]
I don't know. I can only speak for myself.
What do the others think? Is RSS useful or not?
@air2310 Are you familiar with RSS, for example?

**Comment by aswinnarayanan** [comment]
We can remove the big RSS icon from the blog page, and add it at the bottom next to the twitter button.

![image](https://user-images.githubusercontent.com/25297584/134614243-d3779cc8-8020-4fd1-a895-ca10536c3e98.png)

So users accessing the page wont click it by mistake. But existing RSS users can use it if they wish
The RSS feed is managed automatically by the server, so no harm leaving it.

We can add the mailing list option what that is ready

**Reply by aswinnarayanan** [reply]
@civier Done

**Reply by stebo85** [reply]
Love it :) Nicely done @aswinnarayanan.

**Comment by air2310** [comment]
Hi @civier, sorry for the late response to these notifications - I work on a different postdoc in the second half of the week. It sounds like we've reached a good compromise though. 
My two cents: I think the only really important news users need to be kept abreast of is of new releases and any bugs/issues that have been dealt with in these. Perhaps the easiest and most efficient way to implement this would be to set our release history page as the homepage on the browser in neurodesktop. That way anyone who is actually using the platform has the information easily accessible without having to add to their inbox or news-stream clutter.
